src/anndata/_core/aligned_mapping.py

```diff
@@ -265,7 +265,7 @@ def to_df(self) -> pd.DataFrame:
 
     def _validate_value(self, val: Value, key: str) -> Value:
         if isinstance(val, pd.DataFrame):
-            raise_value_error_if_multiindex_columns(val)
+            raise_value_error_if_multiindex_columns(val, f"{self.attrname}.{key}")
```

Comment:
Isn’t this more like

```suggestion
            raise_value_error_if_multiindex_columns(val, f"{self.attrname}[{key!r}]")
```

Reply:
Sure, I'll quotes since it is more like what people write.

Reply:
Ah, the repr does that ok.